libnixf/src/Basic/diagnostic.py

```diff
@@ -192,19 +192,19 @@ class Diagnostic(TypedDict):
     {
         "sname": "sema-extra-rec",
         "cname": "ExtraRecursive",
-        "severity": "Hint",
+        "severity": "Warning",
         "message": "attrset is not necessary to be `rec`ursive",
     },
     {
         "sname": "sema-extra-with",
         "cname": "ExtraWith",
-        "severity": "Hint",
+        "severity": "Warning",
         "message": "unused `with` expression",
     },
     {
         "sname": "sema-escaping-with",
         "cname": "EscapingWith",
-        "severity": "Hint",
+        "severity": "Warning",
```

Comment:
I think we should keep `sema-escaping-with` to `"Hint"`, this diagnostic seems to be too pedantic.